### PR TITLE
PowerShell ISE refresh Pt 2

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary
This is the second part to the ISE conceptual docs refresh for formatting.

## PR Context
Fixes [AB#1660352](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1660352)
Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
+1. **Command breakpoint.** The script pauses whenever the designated command is about to be run
+   during the operation of the script. It can include parameters to further filter the breakpoint to
+   only the operation you want. The command can also be a function you created.
 
-Of these, in the Windows PowerShell ISE debugging environment, only line breakpoints can be set by using the menu or the keyboard shortcuts. The other two types of breakpoints can be set, but they are set from the Console Pane by using the [Set-PSBreakpoint](https://technet.microsoft.com/library/88d2d9ad-17dc-44ae-99aa-f841125b9dc8) cmdlet. This section describes how you can perform debugging tasks in Windows PowerShell ISE by using the menus where available, and perform a wider range of commands from the Console Pane by using scripting.
+Of these, in the Windows PowerShell ISE debugging environment, only line breakpoints can be set by
+using the menu or the keyboard shortcuts. The other two types of breakpoints can be set, but they
+are set from the Console Pane by using the [Set-PSBreakpoint](https://technet.microsoft.com/library/88d2d9ad-17dc-44ae-99aa-f841125b9dc8)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md]: These old links for cmdlets need to be updated throughout each of the updated files.

File: reference/docs-conceptual/components/ise/How-to-Write-and-Run-Scripts-in-the-Windows-PowerShell-ISE.md [hunk 2]
```diff
@@ -149,8 +150,8 @@ Press **CTRL+S** or, on the toolbar, click the **Save** icon, or on the **File**
 
 ### To save a script in ASCII encoding
 
-By default, Windows PowerShell ISE saves new script files (.ps1), script data files (.psd1), and
-script module files (.psm1) as Unicode (BigEndianUnicode) by default.Â To save a script in another
+By default, Windows PowerShell ISE saves new script files (`.ps1`), script data files (`.psd1`), and
+script module files (`.psm`1) as Unicode (BigEndianUnicode) by default.Â To save a script in another
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/How-to-Write-and-Run-Scripts-in-the-Windows-PowerShell-ISE.md]: Looks like the 1 got left outside the codefence.  There's also a weird A on this line.

File: reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md [hunk 3]
```diff
@@ -34,16 +37,22 @@ Supported in Windows PowerShell ISE 3.0 and later, and not present in earlier ve
...
+> environment from which you invoked the command. If the command takes longer to run than the
+> **millesecondsTimeout** value specifies, then the command fails with an exception: "The operation
+> has timed out."
 
 **Script** - System.Management.Automation.ScriptBlock or String
 The script block to run.
 
-**\[useNewScope\]** -  Optional Boolean that defaults to **$true**
-If set to **$true**, then a new scope is created within which to run the command. It does not modify the runtime environment of the PowerShell tab that is specified by the command.
+**\[useNewScope\]** -  Optional Boolean that defaults to `$true`
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md]: Validate parameter casing.

File: reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md [hunk 4]
```diff
@@ -34,16 +37,22 @@ Supported in Windows PowerShell ISE 3.0 and later, and not present in earlier ve
 Runs the given script in the PowerShell tab.
 
 > [!NOTE]
-> This method only works on other PowerShell tabs, not the PowerShell tab from which it is run. The script block is run and any value that is returned from the script is returned to the run environment from which you invoked the command. If the command takes longer to run than the **millesecondsTimeout** value specifies, then the command fails with an exception: "The operation has timed out."
+> This method only works on other PowerShell tabs, not the PowerShell tab from which it is run. The
+> script block is run and any value that is returned from the script is returned to the run
+> environment from which you invoked the command. If the command takes longer to run than the
+> **millesecondsTimeout** value specifies, then the command fails with an exception: "The operation
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md]: Validate parameter casing.

File: reference/docs-conceptual/components/ise/object-model/The-PowerShellTabCollection-Object.md [hunk 5]
```diff
@@ -41,7 +44,8 @@ $psISE.PowerShellTabs.Remove($newTab)
 
 Supported in Windows PowerShell ISE 2.0 and later.
 
-Selects the PowerShell tab that is specified by the **psTab** parameter to make it the currently active PowerShell tab.
+Selects the PowerShell tab that is specified by the **psTab** parameter to make it the currently
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/object-model/The-PowerShellTabCollection-Object.md]: Validate parameter casing.

File: reference/docs-conceptual/components/ise/How-to-Write-and-Run-Scripts-in-the-Windows-PowerShell-ISE.md (hunk 2 above)
Comment (chasewilson) [reply reference/docs-conceptual/components/ise/How-to-Write-and-Run-Scripts-in-the-Windows-PowerShell-ISE.md]: fixed

File: reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md]: forgot all about these. Thanks

File: reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md (hunk 3 above)
Comment (chasewilson) [reply reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md]: Not params in this case Object methods and casing appears correct. if there is something I'm missing I'm open to it:)

File: reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md]: Fixed

File: reference/docs-conceptual/components/ise/object-model/The-ISESnippetObject.md [hunk 6]
```diff
@@ -3,9 +3,13 @@ ms.date:  06/05/2017
 keywords:  powershell,cmdlet
 title:  The ISESnippetObject
 ---
+
 # The ISESnippetObject
 
-An **ISESnippet** object is an instance of the Microsoft.PowerShell.Host.ISE.ISESnippet class. The members of the **$psISE.CurrentPowerShellTab.Snippets** collection are all examples of **ISESnippet** objects. The easiest way to create a snippet is to use the [New-IseSnippet&#91;PSITPro5_ISE&#93;](https://technet.microsoft.com/library/0a6339a3-2683-4a8e-8929-90ad9a95c3e0) cmdlet.
+An **ISESnippet** object is an instance of the Microsoft.PowerShell.Host.ISE.ISESnippet class. The
+members of the `$psISE.CurrentPowerShellTab.Snippets` collection are all examples of
+**ISESnippet** objects. The easiest way to create a snippet is to use the [New-IseSnippet&#91;PSITPro5_ISE&#93;](https://technet.microsoft.com/library/0a6339a3-2683-4a8e-8929-90ad9a95c3e0)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/ise/object-model/The-ISESnippetObject.md]: Missed a technet link.

**Pull request merged**